config.go

```diff
@@ -18,6 +18,12 @@ func intEnvConfig(i *int, name string) {
 	}
 }
 
+func int64EnvConfig(i *int64, name string) {
+	if env, err := strconv.ParseInt(os.Getenv(name), 10, 64); err == nil {
+		*i = env
+	}
+}
+
```

Comment:
No need for a custom parser for `int64`. You can parse the ID to int and cast it to `int64` here: https://github.com/imgproxy/imgproxy/pull/603/files#diff-695e64e5ace3a24d0e73f02a3555553fa8df9d9045bf759fb729ef870eb62c61R54